pkg/rancher-desktop/main/extensions/manager.ts

```diff
@@ -221,7 +223,7 @@ class ExtensionManagerImpl implements ExtensionManager {
 
       tasks.push((async(id: string) => {
         try {
-          return (await this.getExtension(id)).install();
+          return (await this.getExtension(id)).install(allowList);
         } catch (ex) {
           console.error(`Failed to install extension ${ id }`, ex);
```

Comment:
Another earlier-commit comment:

FWICT, `tag ??= undefined;` is there just to get typescript to stop complaining that `tag` is a `let`-var and not a `const`? If so, there should be a comment specifying that. But I would prefer putting this line before the `let` line:
```
// eslint-disable-next-line prefer-const
```